MAINTAINERS.md

```diff
@@ -84,6 +84,44 @@ important maintenance task. When performing the update, follow this procedure:
 1. Create a PR with a title like `Update Envoy to 9753819 (Jan 24th 2021)`,
    describe all performed changes in the PR's description.
 
+## Finding python dependencies
```

Comment:
Great, thank you for documenting this. How often are we going to execute this procedure? It feels a bit disconnected now, since it isn't listed form the regular update procedure, so it might not be run at all?

Reply:
I agree. Wanted to figure out an appropriate cadence for that.  It feels unrelated to the envoy update, so including it there didn't feel like the right place, plus I think weekly is probably overkill for this procedure.

I'm not sure if this is out of scope for the current work, but could we have an issue auto-created once a month?  Or could we have something automated run the commands I created?

Reply:
I lean towards including it in the regular update actually. How about we include a step to do this if it wasn't done for the last 4 weeks? We can include a timestamp in the file and have the person doing the regular update check how old it is.

Reply:
I did my best to include this idea here. I made it within the month instead of last 4 weeks, which I think is just visually easier, but very similar.  Please take a look